docs/hub/datasets-image.md

```diff
@@ -0,0 +1,123 @@
+# Image Dataset
+
+This guide will show you how to configure your dataset repository with image files.
```

Comment:
Maybe we can still link to https://huggingface.co/docs/datasets/image_dataset somewhere for advanced / programmatic use.

Reply:
Hmmm, not sure. The additional content would be the dataset script, and we don't mention it in the rest of the docs here, so... I would leave it unchanged

(edit: we have one mention of the script, I will remove it)